### ISSUE TITLE
Previous link does not disappear on the atom feed at the feed head

### ISSUE THREAD

**Issue by jen20** [EventStore/EventStore#673]
To reproduce (note `paged-feeds-1` has 114 events as confirmed by jq):

```
$ jq '. | length' paged-feeds-1.json
114
```

Commands:

```bash
curl -i -d @paged-feeds-1.json -H "Content-Type: application/vnd.eventstore.events+json" http://127.0.0.1:2113/streams/pagedfeed-146
# after a few more ommitted here
curl -i -H "Accept: application/json" http://127.0.0.1:2113/streams/pagedfeed-146/114/forward/20
```

Output:

```http
HTTP/1.1 200 OK
Access-Control-Allow-Methods: GET, OPTIONS
Access-Control-Allow-Headers: Content-Type, X-Requested-With, X-Forwarded-Host, X-PINGOTHER, Authorization, ES-LongPoll, ES-ExpectedVersion, ES-EventId, ES-EventType, ES-RequiresMaster, ES-HardDelete, ES-ResolveLinkTo, ES-ExpectedVersion
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: Location, ES-Position
Cache-Control: max-age=0, no-cache, must-revalidate
Vary: Accept
ETag: "113;-43840953"
Content-Type: application/json; charset=utf-8
Server: Mono-HTTPAPI/1.0
Date: Wed, 09 Sep 2015 16:30:59 GMT
Content-Length: 938
Keep-Alive: timeout=15,max=100

{
  "title": "Event stream 'pagedfeed-146'",
  "id": "http://127.0.0.1:2113/streams/pagedfeed-146",
  "updated": "0001-01-01T05:00:00Z",
  "streamId": "pagedfeed-146",
  "author": {
    "name": "EventStore"
  },
  "headOfStream": false,
  "links": [
    {
      "uri": "http://127.0.0.1:2113/streams/pagedfeed-146",
      "relation": "self"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/pagedfeed-146/head/backward/20",
      "relation": "first"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/pagedfeed-146/0/forward/20",
      "relation": "last"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/pagedfeed-146/113/backward/20",
      "relation": "next"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/pagedfeed-146/114/forward/20",
      "relation": "previous"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/pagedfeed-146/metadata",
      "relation": "metadata"
    }
  ],
  "entries": []
}
```

The previous link should not be there.

**Comment by sbellware** [comment]
Yes, we see this as well. (@ntl found it in our shop when working on our HTTP client.)

**Comment by jen20** [comment]
This was fixed in version 3.2.1 of Event Store.
